README.md

```diff
@@ -133,6 +142,7 @@ All of these CSS properties are supported. You can pass either a string or a num
 - `clear`
 - `color`
 - `columnGap`
+- `content`
```

Comment:
After pulling in this change to Evergreen locally and swapping out glamor, the Overlay component wasn't showing the transparent grey background due to the missing `content: "";` rule. Since this isn't directly related to animations/keyframes, I can revert this change and make a separate PR for it if desired.